sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -796,3 +797,71 @@ case class SchemaOfJson(
 
   override def prettyName: String = "schema_of_json"
 }
+
+/**
+ * A function which returns all the keys of outer JSON object.
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(json_object) - returns all the keys of outer JSON object.",
+  arguments = """
+    Arguments:
+      * json_object - A JSON object. If it is an invalid string, the function returns null.
+          If it is a JSON array, an illegal argument exception will be thrown.
```

Comment:
Could you sync the PR description and this function accordingly? This function description looks too simple and the PR description seems wrong in the part `Analysis Exception`.
```
If invalid json expression is given, NULL will be returned.
If a null string or json array is given, an Analysis Exception is thrown.
If valid json object is given, all the keys of outer object will be returned as array.
For empty json object, empty array is returned.
```

Reply:
Actually PR description is not updated. I will update it. I will update the function description as well.

Reply:
Thanks!